frame/composable-traits/src/staking/mod.rs

```diff
@@ -133,12 +133,13 @@ pub struct RewardPool<AccountId, AssetId, Balance, BlockNumber, DurationPresets,
 
 	// possible lock config for this pool
 	pub lock: LockConfig<DurationPresets>,
+
 	// TODO (vim): Introduce asset ids for financial NFT as well as the shares of the pool
```

Comment:
Let's remove the TODO comments as they are addressed now

Reply:
bc781fa156cd6b8918e008183db87107308cfa2b